utils/dvtm/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
```

Comment:
Surely I am not a lawyer, but since this file is newly introduced this year, the Copyright year should be just 2016?